applications/nokia-edge-network-controller/README.md

```diff
@@ -0,0 +1,47 @@
+SPDX-License-Identifier: Apache-2.0
```

Comment:
'#' is missing here.
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2022 Nokia Corporation

Reply:
I've put the license and copyright info in text block, # would make it a very large header